k8s/qa/curator.yaml

```diff
@@ -0,0 +1,115 @@
+# Kubernetes deployment file for the Global Health service on AWS.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: curator-qa
+  labels:
+    environment: qa
+    app: curator
+spec:
+  selector:
+    matchLabels:
+      environment: qa
+      app: curator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        environment: qa
+        app: curator
+    spec:
+      containers:
+        - name: curator
+          image: ghcr.io/globaldothealth/list/curatorservice:main
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 3001
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: 3001
+            initialDelaySeconds: 10
+            periodSeconds: 10
+          env:
+            # TODO: Remove this pending https://github.com/globaldothealth/list/issues/961.
```

Comment:
sorry, but you drew the short straw: #961 has been closed for a while, this TODO can be DID. Unless we have other reasons for wanting this output (I think it gets in the way of grokking application logs, personally)